### ISSUE TITLE
Chen does not rhyme with fun

### ISSUE THREAD

**Issue by yegle** [SteveMCarroll/PronouncingNamesForEnglishSpeakers#2]
Chen (陈) does not rhyme with fun, instead it's more close to rhyme with London.

I think you were mistaken with Cheng (程) which sounds more close to fun.

**Comment by SteveMCarroll** [comment]
Here's the (English language) Wikipedia Standard Chinese Phonology chart on this one:
![image](https://user-images.githubusercontent.com/12647923/126930658-dfa33e6e-f8a8-448c-8970-602950cc6aed.png)

for words ending in the sound -en, they notate it in IPA with the neutral [schwa](https://en.wikipedia.org/wiki/Schwa) vowel [ən].  

In the first dictionary (Merriam-Webster) I checked (and my own accent which is a midwestern variant on Standard American English), fun is listed as: fun [fən].   

Turns out that in Oxford Learner's the pronunciation is /fʌn/.   so this is a UK vs US thing.
I'm open to changing this to a word that is the same in both accents, but I would definitely prefer a one syllable one to a two syllable one.   In "London", Oxford Learner's lists the first vowel (/ˈlʌndən/) as the same as fun, and the second vowel as the schwa so I feel like that word doesn't bring the desired clarity.   let me think about this one and find a better word.
